public/js/utils/path.js

```diff
@@ -1,20 +1,2 @@
-function basename(path) {
-  return path.split("/").pop();
-}
-
-function dirname(path) {
-  const idx = path.lastIndexOf("/");
-  return path.slice(0, idx);
-}
-
-function isURL(str) {
-  return str.indexOf("://") !== -1;
-}
-
-function isAbsolute(str) {
-  return str[0] === "/";
-}
-
-module.exports = {
-  basename, dirname, isURL, isAbsolute
-};
+const path = require("path-browser-wrapper");
+module.exports = path;
```

Comment:
Okay. I'll do that then.
On Tue, Aug 30, 2016 at 10:03 PM Jason Laster <notifications@github.com>
wrote:

> In public/js/utils/path.js
> <https://github.com/devtools-html/debugger.html/pull/631#discussion_r76913463>
> :
>
> > -  return path.slice(0, idx);
> > -}
> > -
> > -function isURL(str) {
> > -  return str.indexOf("://") !== -1;
> > -}
> > -
> > -function isAbsolute(str) {
> > -  return str[0] === "/";
> > -}
> > -
> > -module.exports = {
> > -  basename, dirname, isURL, isAbsolute
> > -};
> > +const path = require("path-browser-wrapper");
> > +module.exports = path;
>
> I'd switch this and use the library directly where it is called.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/devtools-html/debugger.html/pull/631/files/8f7f9c8e5e3e87cdbdfb6d1c0cdb5ad3b6079e1e#r76913463>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACXb0e9dNidNcHIL3INzfpTuUF_8P4Krks5qlOD8gaJpZM4JxJlg>
> .
>